### DISCUSSION TITLE
[Feature Request] Code Explanation

### DISCUSSION THREAD

**Discussion by B-R-P** [community/community#7509] [Copilot Conversations] [answered]
I noticed, while writing commenting after writing code it often predicts correctly,
Thus I think a code explanation feature can be added
### Working
After selecting a block of code, pop message('Explain code') shows. 
On clicking it new window opens with explanation in natural language.

It helps to understand complex code easily.

**Comment by MulleDK19** [comment]
It already can. Just let it complete comments..

**Reply by B-R-P** [reply]
Explaining large chunk of code will be great

**Comment by ghost** [comment, marked as answer]
this is now possible in the current copilot labs :)
